Mathlib/Topology/Algebra/Module/Alternating/Topology.lean

```diff
@@ -24,6 +24,22 @@ namespace ContinuousAlternatingMap
 variable {𝕜 E F ι : Type*} [NormedField 𝕜]
   [AddCommGroup E] [Module 𝕜 E] [TopologicalSpace E] [AddCommGroup F] [Module 𝕜 F]
 
+section IsClosedRange
+
+variable [TopologicalSpace F] [TopologicalAddGroup F]
+
+instance instTopologicalSpace : TopologicalSpace (E [⋀^ι]→L[𝕜] F) :=
+  .induced toContinuousMultilinearMap inferInstance
+
+lemma isClosed_range_toContinuousMultilinearMap [ContinuousSMul 𝕜 E] [T2Space F] :
+    IsClosed (Set.range (toContinuousMultilinearMap : E [⋀^ι]→L[𝕜] F →
+      ContinuousMultilinearMap 𝕜 (fun _ : ι ↦ E) F)) := by
```

Comment:
I'm surprised that this is parsed as `(E [⋀^ι]→L[𝕜] F) → ContinuousMultilinearMap` and not `E [⋀^ι]→L[𝕜] (F → ContinuousMultilinearMap)` as would be the case for instance for continuous linear maps or virtually anything else. Is there a priority to fix here? In any case, adding the parentheses to emphasize the parsing would be a good idea, even if Lean is not confused.